e2e_test/streaming/aggregate/approx_percentile.slt

```diff
@@ -0,0 +1,71 @@
+# Single phase approx percentile
+statement ok
+create table t(p_col double, grp_col int);
+
+statement ok
+insert into t select generate_series, 1 from generate_series(1, 1000);
+
+statement ok
+flush;
+
+query I
+ select
+     approx_percentile(0.5, 0.01) within group (order by p_col) as p50,
+     approx_percentile(0.9, 0.01) within group (order by p_col) as p90
+ from t group by grp_col;
+----
+497.7794014558068 907.0313473737922
+
+query I
+select percentile_disc(0.5) within group (order by p_col) as p50, percentile_disc(0.9) within group (order by p_col) as p90 from t group by grp_col;
+----
+500 900
+
+statement ok
+create materialized view m1 as
+ select
+     approx_percentile(0.5, 0.01) within group (order by p_col) as p50,
+     approx_percentile(0.9, 0.01) within group (order by p_col) as p90
+ from t group by grp_col;
+
+query I
+select * from m1;
+----
+497.7794014558068 907.0313473737922
+
+# statement ok
+# recover;
+
+# wait recovery
+# sleep 10s
+
+query I
+select * from m1;
+----
+497.7794014558068 907.0313473737922
+
+# Test state encode / decode
+statement ok
+recover;
```

Comment:
Everythiing passes when this line is removed.
That's because this will trigger the `encode/decode` branch which has some bug, as mentioned here: https://github.com/risingwavelabs/risingwave/pull/17814/files#r1691073080

Reply:
Is this fixed?

Reply:
Yes.